hugo/assets/scss/components/article.scss

```diff
@@ -55,8 +35,49 @@
         word-break: break-word;
     }
 
+    &__info {
+        align-items: center;
+        display: flex;
+        flex-wrap: wrap;
+        gap: 0.75rem $p-gutter;
+        padding-bottom: $p-gutter;
+        width: 100%;
+
+        &::after {
+            border-bottom: 1px solid $c-lavender--light;
+            bottom: 0;
+            content: '';
+            height: 1px;
+            left: $p-gutter;
+            position: absolute;
+            right: $p-gutter;
+        }
+    }
+
     &__content {
-        padding-bottom: 2rem;
+        @include container($w-content--small);
+
+        overflow-wrap: break-word;
+        padding: 0 $p-gutter 2rem;
+    }
+
+    &__footer {
+        display: flex;
+        flex-direction: column;
+        gap: 2rem;
+        margin: 0 auto;
+        opacity: 1;
+        padding: 1.5rem $p-gutter;
+
+        &::before {
+            border-top: 1px solid $c-lavender--light;
```

Comment:
why not have the border on the __footer directly instead of pseudo elements?

Reply:
Because of paddings and margins etc.,  and the border would be too wide. See example below. The red one is when the border-top would be added directly to the `__footer`
![Screenshot 2023-08-23 at 11 40 33](https://github.com/cue-lang/cuelang.org/assets/122522301/6e793bea-9c08-462a-afd9-13dfbb21d7b7)